### ISSUE TITLE
Gunicorn in the admin container does not pass the X-Forwarded-Proto header to Flask

### ISSUE THREAD

**Issue by verdel** [Mailu/Mailu#1740]
### Environment
 - [x] docker-compose
 - [ ] kubernetes
 - [ ] docker swarm

### Versions
I use version 1.7

## Description
I use a scheme with Nginx that handles SSL connections. This Nginx stands in the schema above the Mailu stack. The `TLS_FLAVOR` environment variable has the value `mail`.

When I access endpoint `/admin` using the `https` scheme, I end up getting a redirect to `/admin/ui/`, but with the `http` scheme specified.

This is because gunicorn doesn't allow to handle set secure headers by [default](https://docs.gunicorn.org/en/stable/settings.html#forwarded-allow-ips).
Secure headers are:
 - X-Forwarded-Protocol
 - X-Forwarded-Proto
 - X-Forwarded-SSL

## Options for solving the problem
 - **Less Secure**: Add `--forwarded-allow-ips='*'` to gunicorn execution command [here](https://github.com/Mailu/Mailu/blob/6518ef19af92730fd99b9c148d279a971b856cd8/core/admin/start.py#L21)
 - **More Secure**: Add `--forwarded-allow-ips=` but with `front` container IP. You can get it using `socrate`, as in `config.py` in the `front` container

**Comment by Diman0** [comment]
It is very common to have a reverse proxy in front of mailu.
I cannot replicate your issue. How did you configure your nginx reverse proxy?
I used as reverse proxy the swag image of [linuxserver.io](https://docs.linuxserver.io/images/docker-swag).

This is the site config I used. Maybe this can help you configuring nginx:
```
# redirect all traffic to https
server {
        listen 80 default_server;
        listen [::]:80 default_server;
        server_name _;
        return 301 https://$host$request_uri;
        add_header Strict-Transport-Security "max-age=15552000; includeSubDomains" always;
}

#drop traffice to mydomain.com
server {
    listen 443 ssl;
    listen [::]:443 ssl;

    server_name _;
    return      444;

    include /config/nginx/ssl.conf;
}

#mailu
server {
    listen 443 ssl;
    server_name mydomain.com;
    add_header Strict-Transport-Security "max-age=15552000; includeSubDomains" always;
    include /config/nginx/ssl.conf;
    
    client_max_body_size 0;

    location /webmail {
        include /config/nginx/proxy.conf;
        resolver 127.0.0.11 valid=30s;
        proxy_pass https://front:443/webmail;
    }

    location /admin {
        include /config/nginx/proxy.conf;
        resolver 127.0.0.11 valid=30s;
        proxy_pass https://front:443/admin;
    }

}

```
The config files (ssl.conf) can be read in the [github repository](https://github.com/linuxserver/docker-swag/tree/master/root/defaults)

**Comment by Diman0** [comment]
This will be addressed by PR #1765.

**Comment by Diman0** [comment]
PR1765 has been merged. I'm closing this issue.
